lib/SILOptimizer/Mandatory/DiagnoseStaticExclusivity.cpp

```diff
@@ -1114,22 +924,24 @@ static void checkAccessedAddress(Operand *memOper, StorageMap &Accesses) {
   }
 
   // Strip off address projections, but not ref_element_addr.
-  SILValue base = findAccessedAddressBase(address);
-  // `base` is null for address producers that are only used for local
-  // initialization.
-  if (!base || !isPossibleFormalAccessBase(base))
+  const AccessedStorage &storage = findAccessedStorage(address);
+  // `storage` is invalid for address producers that are only used for local
+  // initialization, hopefully.
```

Comment:
... hopefully?

Reply:
As in, hopefully someone eventually reviews all of this. New comment:
```
  // findAccessedStorage may return an invalid storage object if the address
  // producer is not recognized by its whitelist. For the purpose of
  // verification, we assume that this can only happen for local
  // initialization, not a formal memory access. The strength of
  // verification rests on the completeness of the opcode list inside
  // findAccessedStorage.
```
In the subsequent code, I also replaced all occurrences of 
`AccessedStorage(address, kind);`
with
`AccessedStorage(address, AccessedStorage::Unidentified);`
for clarity.